apps/dashboard/test/integration/batch_connect/session_contexts_controller_test.rb

```diff
@@ -23,15 +23,13 @@ class BatchConnect::SessionContextsControllerTest < ActionController::TestCase
       # read form.yml and cache.json from the tmp directory
       BatchConnect::App.any_instance.stubs(:version).returns("1.0.0")
       BatchConnect::App.any_instance.stubs(:root).returns(Pathname(tmpdir.to_s))
-      @controller.stubs(:cache_file).returns(Pathname(cache_json.path))
 
       # only the 'new' cluster is enabled (read from clusters.d)
       OodAppkit.stubs(:clusters).returns([
         OodCore::Cluster.new({ id: 'new', job: { some: 'job config' }})
       ])
 
-
-      get :new, params: { token: 'dev/test' }
+      get '/batch_connect/dev/test'
```

Comment:
Can we use a Rails helper here? maybe `batch_connect_path(token: dev/test)`?

Reply:
I added a helper but let me know if it looks like what you intended.

Reply:
Sorry this is what it should be. `new_batch_connect_session_context_path('dev/test')`

It's a rails thing where you can reference the route, this one below, by appending `_path` and specifying arguements (in this case `token`).  It's so we don't have to hard code strings, instead we can utilize the rails infrastructure to tell us the URL.

```
new_batch_connect_session_context GET    /batch_connect/*token/session_contexts/new(.:format)                     batch_connect/session_contexts#new
```

Reply:
Very cool trick to learn. Thanks!